packages/react/src/exchanges/hooks/useExchange.ts

```diff
@@ -27,7 +27,7 @@ function useExchange(
   options: UseExchangeOptions = {},
 ) {
   const exchangeIdHookParameter = exchangeId;
-  const { enableAutoFetch = true, fetchConfig } = options;
+  const { enableAutoFetch = true, fetchConfig, orderItemUuid = '' } = options;
```

Comment:
Since `useExchangeFilters` accepts a undefined value for orderItemUuid, you do not need to provide a default value for it here.

```suggestion
  const { enableAutoFetch = true, fetchConfig, orderItemUuid } = options;
```

Reply:
Done